_posts/2020-07-12-ksf-lead-after-first-weekend.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: news
+title: Kenilworth Sixth Form lead after the first weekend of matches
+---
+
+After the first 24 matches, team KSF from Kenilworth Sixth Form head up the
+league leaderboard with 58 points. They're closely followed by JAMDynamics and
+Lawrence Sheriff School.
+
+The league matches provide a proving ground for all the teams' hard work over
+the past few months in preparing their robot code. There is also a prize for the
+rookie team which scores the best in the league, which the Kenilworth Sixth Form
+and JAMDynamics teams are currently well placed for.
+
+The teams now have a week to work on and improve their code before another 24
+league matches are played over the course of next weekend. As we've seen with
+the changes in the league from Sunday's matches, this means there's still all to
```

Comment:
"Sunday" here feels out of place?

```suggestion
the changes in the league from today's matches, this means there's still all to
```

Reply:
Will this still read right tomorrow later in the week?

Reply:
It will considering the post is reasonably timely. In the same way that in 2 weeks from now, "Sunday" won't be correct.

Reply:
Fair. I guess it is dated anyway.